etc/eagle_config.json

```diff
@@ -32,8 +23,24 @@
             }
         ]
     },
-
+    "capacity": {
+        "enabled":  true,
```

Comment:
What's the point of having parameter enabled? if there is no configuration => it's disabled.